services/workflows-service/src/data-analytics/data-analytics.service.ts

```diff
@@ -39,9 +39,6 @@ export class DataAnalyticsService {
         });
     }
 
-    // Used for exhaustive check
-    inlineRule satisfies never;
```

Comment:
This line is important to verify we didn't miss any function. I think it's better you'll revert this line removal.

Reply:
It just allow TS show the rest of the code as relevant instead of such a gray which it wont get this section 
<img width="746" alt="image" src="https://github.com/ballerine-io/ballerine/assets/6435752/23aaa753-ae27-402b-b42c-9a954acd41b2">

Reply:
with my type changes its looks like that...
<img width="830" alt="image" src="https://github.com/ballerine-io/ballerine/assets/6435752/afa0628a-dd2b-4fd1-b3bf-b7b7dfb4976e">